pages/about.html

```diff
@@ -1,12 +1,73 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About Sana</title>
+    <link href="../styles/style.css" rel="stylesheet">
 </head>
 <body>
-  
+    <header>  
+        <h1>A Little About ME!</h1>
+        <div class="top">
+            <a href="index.html"> Swiftie Website</a>
+            <a href="portfolio.html"> My Portfolio</a>
+        </div>
```

Comment:
Instead of using a div element here, which has no semantic meaning, we should put these 2 links inside a nav element because a nav element represents a section of a page whose purpose is to provide navigation links, either within the current document or to other documents. You can then put the class "top" on the nav element

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav